extensions/database/src/com/google/refine/extension/database/DatabaseModuleImpl.java

```diff
@@ -111,20 +111,18 @@ private Properties loadProperties(File propFile) {
                 if(logger.isDebugEnabled()) {
                     logger.debug("Loading Extension properties ({})", propFile);
                 }
-                BufferedInputStream stream = null;
-                try {
-                     ps = new Properties();
-                    stream = new BufferedInputStream(new FileInputStream(propFile));
+                final BufferedInputStream stream = new BufferedInputStream(new FileInputStream(propFile));
+                try (stream) {
```

Comment:
Why not put the whole variable assignment into the `try ( )`? Like the first example here: https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html

Reply:
it felt "longish", but that's fine.